index.html

```diff
@@ -259,6 +259,25 @@ <h1>Important Terms</h1>
 				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT-SPI]], and <a href="https://wiki.linuxfoundation.org/accessibility/iaccessible2/start">IAccessible2</a> [[IAccessible2]].</p>
 			</dd>
+			<dt><dfn data-export="" data-lt="accessibility child|accessibility children|child element|child|children|child elements">Accessibility child</dfn></dt>
+			<dd>
+				<p>To calculate the <a>accessibility children</a> of an element `X`, first considered a modified <abbr title="Document Object Model">DOM</abbr> tree where elements (and their subtrees) specified by `aria-owns` are reparented to the element with `aria-owns` set.
+				<p>Now, an <a>accessibility child</a> of an <a>element</a> `X` is considered to be any of the following:
+				<ul>
+					<li>A DOM child of `X` in that modified DOM tree.</li>
+					<li>A DOM descendant of `X` with only elements of role <rref>generic</rref> or <rref>none</rref> intervening in that modified DOM tree.</li>
+				</ul>
+				</p>
+                                <p class="note">Accessibility child, accessibility descendant and accessibility parents are terms used to defined required relationships between different DOM elements with ARIA roles or properties. The terms do not refer to a platform accessibility API trees and the relationship between platform accessiblity nodes.</p>
+			</dd>
+			<dt><dfn data-export="" data-lt="accessibility descendant|accessibility descendants">Accessibility descendant</dfn></dt>
+			<dd>
+				<p>To calculate the <a>accessibility descendants</a> of an element `X`, first considered a modified <abbr title="Document Object Model">DOM</abbr> tree where elements (and their subtrees) specified by `aria-owns` are reparented to the element with `aria-owns` set. The <a>accessibility descendants</a> of `X` are all DOM descendants of `X` in that modified DOM tree.</a></p>
+			</dd>
+			<dt><dfn data-export="" data-lt="accessibility parent|parent element|parent">Accessibility parent</dfn></dt>
+			<dd>
+				<p>To calculate the <a>accessibility parent</a> of an element `X`, first considered a modified <abbr title="Document Object Model">DOM</abbr> tree where elements (and their subtrees) specified by `aria-owns` are reparented to the element with `aria-owns` set. The <a>accessibility parent</a> of `X` is the closest DOM ancestor of the `X` in that modified tree that does not have role <rref>generic</rref> or <rref>none</rref>.</p>
+			</dd>
```

Comment:
I moved this section to be more alphabetical. We talked about moving this out of the "important terms" section of this document, @jnurthen had the strongest opinions on this but he is out for 3 weeks! (the PR can be open that long I'd be happy to move this).